qemu/tests/cfg/cpu_topology_test.cfg

```diff
@@ -17,13 +17,41 @@
             no RHEL.6 RHEL.7
             no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
             required_qemu = [8.2.0,)
+            check_core_id_cmd = 'cat /sys/devices/system/cpu/cpu%s/topology/core_id'
     variants:
         - default:
             no s390x
         - siblings_core:
-            only s390x
+            no aarch64
             RHEL.8:
                 check_siblings_cmd = "awk -F'-' '{print $2 - $1 + 1}' /sys/devices/system/cpu/cpu0/topology/core_siblings_list"
```

Comment:
x86 defined this one in L9, this will override it, seems for both RHEL8/9 can use the same one.

btw, for future major releases, we don't have a default one.

Reply:
updated